vibrato/src/dictionary.rs

```diff
@@ -24,6 +24,19 @@ pub use crate::dictionary::word_idx::WordIdx;
 
 pub(crate) use crate::dictionary::lexicon::WordParam;
 
+const VERSION: &str = env!("CARGO_PKG_VERSION");
+const MAGIC_LEN: usize = 20;
```

Comment:
There is no need to determine the length.
It would be sufficient to read the beginning for the length of `MODEL_MAGIC` as follows.
```rust
const MODEL_MAGIC: &[u8] = b"VibratoTokenizer 0.5.0\n";

let mut magic = [0; MODEL_MAGIC.len()];
rdr.read_exact(&mut magic)?;
```
Any termination sign can be used, but `\n` is probably the easiest to understand.
In addition, Vibrato is a very common name, so VibratoTokenizer is better for identification.

Reply:
@vbkaisetsu Okay. I will follow the same manner as Vaporetto.